app/src/androidTest/java/com/example/warriorsocial/ui/login/RegistrationCheckerTest.java

```diff
@@ -0,0 +1,117 @@
+package com.example.warriorsocial.ui.login;
+
+import android.content.Context;
+import android.widget.EditText;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.auth.FirebaseAuth;
+
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RegistrationCheckerTest {
+
+    private static RegistrationChecker registrationChecker;
+    private static User testUser;
+    private static FirebaseAuth fAuth;
+    private static Context context;
+
+    private static EditText testEmail;
+    private static EditText testEmail2;
+    private static EditText testEmail3;
+
+    private static EditText testPassword;
+    private static EditText testPassword2;
+    private static EditText testPassword3;
+    private static EditText testPassword4;
+
+    @BeforeClass
+    public static void before() {
+        // Initiate a context; In this case we need to create a "sample" context for unit testing using InstrumentationRegistry
+        // This is necessary as for one of the methods tested, context is required
+        context = InstrumentationRegistry.getInstrumentation().getTargetContext();
```

Comment:
This looks good. Curious to see if this will run with travis.ci since it is an instrumented test which requires use of an emulator.